### PR TITLE
Bump necoperf v0.1.3

### PR DESCRIPTION
**Pull request by takara9**

### CODE REVIEW

**Review by zeroalphat — COMMENTED** [review]
Can you also update the versions of `Makefile.versions` and `go.mod`?
https://github.com/cybozu-go/necoperf/blob/main/docs/maintenance.md

File: Dockerfile.daemon [hunk 1]
```diff
@@ -18,8 +18,10 @@ ENV HOME=/home/necoperf
 COPY --from=flatcar /bin/ /bin/
 COPY --from=flatcar /sbin/setcap /sbin/setcap
 COPY --from=flatcar /lib64 /lib64
+COPY --from=flatcar /usr/lib64 /usr/lib64
 COPY --from=flatcar /usr/bin/perf /usr/bin/sleep /usr/bin/
 
+
```
Comment (zeroalphat) [inline Dockerfile.daemon:24]: Please delete the unnecessary line breaks.

File: go.mod [hunk 2]
```diff
@@ -1,8 +1,8 @@
 module github.com/cybozu-go/necoperf
 
-go 1.22.0
+go 1.23.0
 
-toolchain go1.22.6
+toolchain go1.23.3
```
Comment (zeroalphat) [inline go.mod:5]: When I ran `go mod tidy`, it caused some changes to appear. 
Could you please run the `go mod tidy -go=1.23.3` command to update the module?

**Review by zeroalphat — CHANGES_REQUESTED** [review]

File: Dockerfile.daemon (hunk 1 above)
Comment (takara9) [reply Dockerfile.daemon:24]: deleted the empty line

**Review by zeroalphat — APPROVED** [review]

**Pull request merged**